\documentclass[11pt,oneside,letterpaper]{article}

% graphicx package, useful for including eps and pdf graphics
\usepackage{graphicx}
\DeclareGraphicsExtensions{.pdf,.png,.jpg}

% basic packages
\usepackage{color} 
\usepackage{parskip}
\usepackage{float}
\usepackage{hyperref}

% text layout
\usepackage{geometry}
\geometry{textwidth=15.25cm} % 15.25cm for single-space, 16.25cm for double-space
\geometry{textheight=22cm} % 22cm for single-space, 22.5cm for double-space

% helps to keep figures from being orphaned on a page by themselves
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}

% bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,font=small]{caption}

% review layout with double-spacing
%\usepackage{setspace} 
%\doublespacing
%\captionsetup{labelfont=bf,labelsep=period,font=doublespacing}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}
%\renewcommand\citeleft{(}
%\renewcommand\citeright{)}
%\renewcommand\citeform[1]{\textsl{#1}}

% Remove brackets from numbering in list of References
\renewcommand\refname{\large References}
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

\usepackage{authblk}
\renewcommand\Authands{ \& }
\renewcommand\Authfont{\normalsize \bf}
\renewcommand\Affilfont{\small \normalfont}
\makeatletter
\renewcommand\AB@affilsepx{, \protect\Affilfont}
\makeatother

% notation
\usepackage{amsmath}
\usepackage{amssymb}
\newcommand{\virus}{\mathbf{x}}						% virus coordinate
\newcommand{\serum}{\mathbf{y}}						% serum coordinate
\newcommand{\viruses}{\mathbf{X}}					% set of virus coordinates
\newcommand{\sera}{\mathbf{Y}}						% set of serum coordinates
\newcommand{\ve}{v}									% virus effect
\newcommand{\se}{s}									% serum effect
\newcommand{\ves}{\mathbf{v}}						% set of virus effects
\newcommand{\ses}{\mathbf{s}}						% set of serum effects
\newcommand{\point}{f_{\scriptscriptstyle \vert}}	% point likelihood
\newcommand{\threshold}{f_{\textstyle \lrcorner}}	% threshold likelihood
\newcommand{\interval}{f_{\sqcup}}					% interval likelihood
\newcommand{\mdssd}{\varphi}						% MDS standard deviation
\newcommand{\virussd}{\sigma_x}						% virus / diffusion standard deviation
\newcommand{\serumsd}{\sigma_y}						% serum standard deviation
\newcommand{\drift}{\mu}							% drift / advection
\newcommand{\tree}{\tau}							% phylogeny
\newcommand{\vn}{n}									% number of viruses
\newcommand{\sn}{k}									% number of sera
\newcommand{\normal}{\mathcal{N}}					% normal distribution
\newcommand{\bwithin}{\beta_w}		% within clade drift coefficient
\newcommand{\bsister}{\beta_s}		% sister clade drift coefficient
\newcommand{\bother}{\beta_t}			% across clade drift coefficient
\newcommand{\incclade}[1]{y_\mathrm{#1}}
\newcommand{\driftclade}[1]{x_\mathrm{#1}}
\setlength{\arraycolsep}{2pt}
\newcommand{\smalltwomatrix}[2]{\scriptsize \Big( \begin{matrix} #1 \\ #2 \end{matrix} \Big)}				% pretty inline matrix 
\newcommand{\smallfourmatrix}[4]{\scriptsize \Big( \begin{matrix} #1 & #2 \\ #3 & #4 \end{matrix} \Big)}	% pretty inline matrix 
\newcommand{\twomatrix}[2]{\left( \begin{matrix} #1 \\ #2 \end{matrix} \right)}								% pretty inline matrix 
\newcommand{\fourmatrix}[4]{\left( \begin{matrix} #1 & #2 \\ #3 & #4 \end{matrix} \right)}					% pretty inline matrix 

\begin{document}

\newgeometry{top=4cm}

Dear eLife editorial board,

Please find attached our extensively revised manuscript entitled ``Integrating influenza antigenic dynamics with molecular evolution''.  This is a resubmission of manuscript number 04-04-2013-RA-eLife-00803. The editorial response stated that:

\begin{quote}
Given, however, the extreme importance of this work to understanding influenza evolution and vaccine selection and the potential to provide a quantitative assessment of the association between drift (as measured by cartography) and incidence, the reviewers and editor would be willing to consider a fresh submission with major revisions that convinced us that the correlation is true or that there is perhaps no correlation between drift and incidence (or conceivably that antigenic cartography is not a good way to measure drift). 
\end{quote}

We have paid particular attention to the reviewer's criticisms regarding our observed correlation between year-to-year antigenic drift and influenza incidence.  We have revised our statistics to show correlation within A/H3N2, A/H1N1, B/Vic and B/Yam individually and as well as to show that measurements of antigenic drift are robust to sampling counts.  Additionally, we have included substantial new results regarding relationships between amino acid substitutions and antigenic drift as suggested by reviewer 2 and provided a new analysis of phylogenetic trees as suggested by reviewer 1.

Point-by-point review responses are attached, as well as a diffed PDF detailing all the changes that have been made since the initial submission.  Additionally, a detailed list of every criticism and exactly what was revised in the manuscript in response to this criticism is available at \url{https://github.com/trvrb/flux/issues?state=closed}.

Sincerely,\\
Trevor Bedford

\restoregeometry

\newpage

\section*{Reviewer responses}

Original reviewer criticisms are in plain text.  Our responses follow in \textbf{bold}.  

%%% REVIEWER 1 %%%
\section*{Reviewer 1}

This paper is exceptional in many ways, but also contains some flaws that need to be addressed by revisions. If these flaws are addressed, I would be highly supportive of publication. This paper is complex, so I have not been able put all of my major critiques in the first 500 words. Instead, I summarize my major critiques here, and then further detail them along with the minor points.

This paper deals with the antigenic evolution of influenza. A foundational paper in Science in 2004 by Smith et al introduced ``antigenic cartography,'' the technique used to monitor antigenic drift and select the influenza vaccine. This paper makes a clear methodological improvement on the original antigenic cartography technique. The new approach integrates phylogenetics, allowing inclusion of viral sequences and isolation dates. This paper also shows that accounting for virus and serum effects improves cartography. As the authors briefly note, the new method is also extensible to evolutionary questions. All of these improvements are made using a freely available phylogenetic software program that is the standard in the field. These methodological improvements would immediately merit publication in a technique-specific journal.

In addition, this paper presents biological results that should interest virologists, public health officials, and evolutionary biologists. While these results do not really alter the current understanding of influenza antigenic evolution, they extend this understanding from H3N2 to other clades, provide a more detailed analysis than has previously been performed for any clade, and quantify factors that have previously been characterized only qualitatively.

I do, however, have some major critiques:

It is difficult to understand the biological results without grasping technical aspects of the computational approach. In my specific comments, I suggest some changes.

The authors need to explain where they got the incidence data, how they chose the date ranges, specify when they are using relative versus absolute incidence, and provide raw data as supporting material. I describe my concerns in more detail in the specific comments.

The improvement from inclusion of serum effects deserves comment and explanation if possible (are the effects due to the serum source)? Table 1 should include a model with virus effects but no serum effects. I describe my concerns in more detail in the specific comments.

MAJOR POINTS IN MORE DETAIL:

I have concerns about Figure 5, which the authors use to support their claim that ``antigenic drift drives incidence rates.'' The authors first establish the well-known fact that H3N2 incidence is highest, followed by H1N1 and then B. They also find that relative incidence is correlated with the rate of antigenic drift. However, this does not imply causality in terms of the higher average drift driving the higher average incidence. The authors do not claim any causality in this correlation between average drift and incidence in the four clades, and I am therefore fine with this part of the analysis. The problems arise in Figure 5A. The authors find a significant correlation between drift and incidence taken over all years and clades, and then use this to argue that drift drives incidence. I would agree with this conclusion if higher drift was associated with higher incidence with a correlation that is better than that obtained simply by comparing the averages of the four clades. But I don't think this is true. In Figure 5A, the authors are averaging over all clades, making the correlation in 5A a trivial consequence of the fact that H3N2 has higher average incidence and drift. Even if the rate of drift and incidence were constant year-over-year for each clade (and simply differed among clades), the authors would observe the correlation that they report. Put another way, imagine that every year the drift and incidence in each clade was equal to its average. Then Figure 5A would in effect correspond to simply recalculating the correlation between average drift and incidence in a scenario where each point is included as many times as there are years being examined. The more years that were included the higher the P-value would get because the number of data points would increase, but really the single actual trend (that H3N2 is higher in average drift and incidence) is just being counted multiple times. In order to draw a conclusion stronger that drift drives incidence, the authors need to show that incidence and drift are correlated within each clade individually, or else standardize the incidence so that the mean and variance for each individual clade is zero and one. It is not clear to me that there is any correlation between the drift and incidence within any of the individual clades. A similar problem affects Figure 5B - for example, since H3N2 has the highest drift, it will always tend to be associated with lower drift in other clades since the other clades inherently have lower average drift. But this does not imply direct interference between clades. I am open to further discussion about this point, but I do not see how Figure 5 implies that antigenic drift drives incidence, or that there is dynamical interference across clades. By the way, I do not think that a lack of correlation would be a disqualifying concern in terms of publication. Finding no evidence that antigenic drift drives incidence is just as important as finding that it does. But the claim needs to be correct.

\textbf{This is a very astute criticism.  We see how differing overall incidence between clades could have given an artifactual signal in our previous year-to-year drift vs incidence comparisons.  In this revision, we have reanalyzed the data in the way suggested; we show that year-to-year antigenic drift and incidence are correlated within each clade individually, arriving at correlation coefficients of 0.51, 0.29, 0.44 and 0.14 for A/H3N2, A/H1N1, B/Vic and B/Yam respectively.  None of these correlations are significant on their own, however observing four correlation coefficients of this magnitude is highly unlikely under a null model derived from bootstrap permutations ($p = 0.018$).  Because the increase in incidence tends to follow periods of pronounced antigenic drift, we conclude that there appears to a be causal relationship between antigenic change and increased incidence.}

\textbf{However, in redoing this analysis on a lineage-by-lineage basis, we lost much of the signal for interference between lineages.  We think there may still be something there, but the nuanced analysis that this issue deserves seems beyond the scope of the paper.  We have decided to instead drop the discussion of interference between lineages.}

The incidence data are poorly explained. In the first paragraph of the Epidemiological consequences of antigenic drift section, the authors use relative incidence. In the next paragraph, are they switching to absolute incidence? This is not clear. If they are switching to absolute incidence, I don't think they ever explain in the Methods where that data comes from. In either case, what are the justifications for using absolute versus relative incidence? 

\textbf{In revising this section in the manuscript, we have made the source of the incidence data and its construction much more clear.  In addition, we have removed the relative vs absolute incidence confusion, sticking instead with one measure: ILI $\times$ proportion of viral isolates attributable to a lineage.}

Also, why do they not analyze incidence prior to 1998/1999? 

\textbf{The 1998/1999 season is the earliest we have that properly distinguishes B/Vic and B/Yam in the CDC data.}

Finally, because each influenza season spans two calendar years, why do they not break up the data by season rather than year?

\textbf{This would be much preferable.  However, many virus isolates used in this analysis lack temporal resolution beyond the year of sampling.  Because of this, we decided to stick with just year of isolation for the present analysis.}

An interesting result is that estimating the serum effects improves the performance a great deal (more than including the phylogenetic data, in fact!). It suggests that there may be systematic differences between the sera. Can this be attributed to the serum itself (such as whether it is goat versus ferret), or to the particular lab or study? Could the authors correlate the estimated serum effects back to the originating source for that serum to check? 

\textbf{We would suggest that a serum with a larger effect contains a more concentrated and active set of antibodies than a serum with a smaller effect. The increased number of antibodies could be due to experimental variation in serum extraction and processing or due to variation in immune response and timing between ferrets, i.e. some ferrets may mount an immediate and strong immune response, while others may mount a weaker response.  This causes variation in overall strength of the serum.  We try to control for the overall strength when looking at patterns of cross-reactivity through the use of `serum effects'.  Some differences in serum effects could be tracked down to particular studies, but this work is beyond the scope of the current study.  Here, we've essentially treated serum effect as a nuisance parameter.  We've revised the text to discuss this as well as describe why estimating serum effects improves model performance over fixing serum effects.}

As a related point, I was surprised that inclusion of the virus effects led to only minor improvement, since prior work (http://www.ncbi.nlm.nih.gov/pubmed/19900932) suggests that some viruses are generally more resistant to HI than others. However, the virus effects are only included in a model (\#9 in Table 1) where the serum effects are already included, and it is possible to imagine that including just one of these terms somehow helps correct for both effects. Can the authors include a tenth model in Table 1 that includes the virus effects but not the serum effects? This would clarify if serum effects are actually more important than virus effects, or simply whether inclusion of either one individually is more helpful than both together.

\textbf{The inclusion of `virus effects' was meant to capture exactly this observation, that some viruses may be generally more resistant to HI. To further assess their importance, we followed this suggestion and included test error for a model that estimates virus effects, but leaves serum effects fixed at maximum titers. In this case, we observe similar, but not quite as substantial, improvements to test error as the model that estimates serum effects, but does not include virus effects.  From this observation, we conclude that estimating either virus or serum effects both improve the model.}

IMPORTANT BUT NOT SCIENTIFICALLY SUBSTANTIVE POINTS: 

The authors should be commended for making publicly available all of the HI data that they have assembled (a good scientific practice that sadly has not been adhered to in some previous publications on antigenic cartography). However, they should mention in the text the availability of these data on datadryad - I almost missed it since it is only mentioned on the cover page. They could also consider more clearly specifying where the code relevant to this paper is found in the BEAST repository.

The one exception to the aforementioned praise on data availability is the influenza incidence data - I could not find any tables providing these data. They need to be provided.

\textbf{We apologize for this oversight.  The incidence data table used in our analysis has now been included with the Dryad archive and a link to this archive included under availability.  We've also included a reference to the BEAST tutorials on implementing these cartographic models.}

The authors need to define their nomenclature regarding clade / lineage / type / strain. For instance, they say: ``We find that 1 unit of antigenic drift within a clade increases incidence by 1.26 standard deviations, while 1 unit of drift in a clade of the same type decreases incidence by 0.61 standard deviations and 1 unit of drift in clades of differing type decreases incidence by 0.32 standard deviations. The term for across lineage Bt is small in magnitude.'' I had to read this many times before concluding that type refers to A or B, clade to refers to any of the four classifications. I am still not sure if lineage has a specific meaning or is simply a redundant word for clade.

\textbf{We've cleaned up our language regarding clades and lineages in the text.  We consider A/H3N2, A/H1N1, B/Vic and B/Yam to each represent separate \textit{lineages} of influenza.  We no longer refer to separate \textit{clades}.}

The comparison in Table 1 is crucial, but the model descriptions are very hard to understand until after reading the Methods carefully. The difficulty is because the models are described in technical terms (dimensionality, location prior, serum effects, virus effects). The most confusing part is the location prior. My understanding (and if this is wrong then things really need to be clarified) is that uninformed refers to making the predictions just from HI data, drift refers to making the predictions from HI data + date of isolation, and diffusion / drift refers to making the predictions from HI data + date of isolation + phylogenetic relationship. If the models were named in terms of the data used to make the predictions rather than in terms of the prior, it would be easier to understand intuitively. For instance, a reader could then simply see, ``OK, they included the isolation date in the model and the predictions improved by this much.'' I recognize that the prior plays a role in addition to the data, but I feel that a more intuitive naming scheme should be considered with detailed descriptions in the Methods. There also might be a more intuitive way to denote serum effects and virus effects, although this is less confusing than the location prior.

\textbf{This is a very helpful suggestion.  We've included a new column in Table 1 labeled `Data' that specifies the combination of data sources that go into the model (HI vs HI/year vs HI/year/seq).  We've also clarified that serum and virus effects act to give a baseline expectation for titer when virus and serum have identical antigenic locations.}

The first figure of the paper (Figure 1) is basically methodological rather than presenting a biological result. Although the data in this figure may be important, making it the first thing the reader encounters makes the paper less accessible to readers primarily interested in the biological results.

\textbf{We agree and have revised the manuscript to remove this figure.  Instead of the figure we report Pearson's correlations between antigenic distance and error ($r = 0.098$) and between year and error ($r = -0.007$).}

The authors suggest that interference between clades could be due to CTL epitopes. While this is one plausible explanation, it could also be due to other forms of cross-reactive immunity such as broadly reactive anti-hemagglutinin antibodies (http://www.ncbi.nlm.nih.gov/pubmed/21878571, http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2860935/) or antibodies against internal proteins (http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3597515/).

\textbf{We have expanded this paragraph to encompass broadly neutralizing anti-HA antibodies and antibodies to internal proteins and included the above references.}

MINOR COMMENTS: 

The first paragraph says: ``A large proportion of the disease burden of influenza stems from antigenic drift; it is why vaccines remain only transiently effective.'' In fact, influenza vaccines are probably relatively ineffective for a number of reasons, and there is no correlation between vaccine match and efficacy (although the data is admittedly noisy), as discussed on page 28 of http://www.cidrap.umn.edu/cidrap/files/80/ccivi\%20report.pdf So it might be more accurate to say that ``antigenic drift is probably one reason why influenza vaccines are only transiently effective.''

\textbf{We have revised this paragraph to make it clear that antigenic drift causes efficacy to decline over time, but that there are other factors that generally limit influenza vaccine efficacy.}

In the second paragraph of the Introduction, the authors refer to ``more evolved viruses.'' Because the concept of directionality in evolution is poorly defined, ``more recent'' would be better than ``more evolved.''

\textbf{We agree. This has been corrected.}

In the third paragraph, the authors refer to ``largely linear movement'' of the virus. Although such a description becomes meaningful later in the paper when the authors start specifying antigenic dimensions, at this point in the introduction it is unclear what the virus is moving through in a linear fashion.

\textbf{We concur and have revised this sentence.}

The authors say, ``If a single map is desired, then the MDS method may be preferred while the BMDS method provides flexibility, allowing other sources of information, such as genetic data, to be more easily incorporated.'' Is this the first part of this sentence true? The BMDS method introduced in this paper is such a clear improvement on the original method in the 2004 Smith et al Science paper that this new approach should always be preferred. I don't see any difficulties with summarizing the posterior of the BMDS as done in this paper. It might be possible to make some non-Bayesian (such as maximum-likelihood version) of the method in this paper that included genetic data / dates / virus+serum effects, and then argue that such a maximum-likelihood version could be preferred to the BMDS method here. But if the comparison is simply with the 2004 Smith et al Science approach (which only uses HI data without dates / phylogeny / virus+serum effects), then the current approach is clearly better due to the inclusion of additional data, regardless of one's philosophical preference for Bayesian versus maximum likelihood.

\textbf{We agree that we may have been too hesitant in promoting the advances included in the BMDS method.  To clarify this point, we've separated the comparison of ML vs Bayes from the comparison of the 2004 Smith et al. cartographic model and the new model that includes virus effects, serum effects and evolutionary priors.  MDS vs BMDS is in some ways a matter of preference; MDS will be faster and have the convenience of giving a single map to work with, while BMDS will be slower to produce a full posterior sample and this sample will be less convenient to work with than a single set of points.  We did find, however, the model extensions to be quite useful in the analysis of this data.}

In the text, the authors introduce the three beta coefficients without first defining what they mean in either intuitive or mathematical terms.

\textbf{We have revised the text to include a more thorough introduction to the linear model that better describes the meaning of the beta coefficients.}

The authors say, ``we find that interference between sister clades, e.g. B/Vic and B/Yam, appears stronger than interference between more divergent clades, e.g. B/Vic and A/H3N2.'' However, this statement is not linked to any analysis/data - where is the figure or table?

\textbf{This statement comes from the relative magnitudes of the $\beta_s$, representing the effect of antigenic drift on sister clades, and $\beta_t$, representing the effect of drift on more divergent clades.  We find $\beta_s = -0.61$ and $\beta_t = -0.32$.  We have clarified the text accordingly.}

Could the authors elaborate in the Methods about the meaning of the lines connecting points in Figure 2 and 3? They say these represent ``mean posterior diffusions paths,'' but I'm having trouble grasping what that means.

\textbf{We apologize for the lack of clarity here.  We have included an additional paragraph in Methods under Implementation that describes the algorithm to estimate ``mean diffusion paths''.  This paragraph reads as follows:}

\textbf{``We summarize diffusion paths of viral lineages (Figure~2, Figure~3) by taking each virus and reconstructing $x$ and $y$ locations along antigenic dimensions 1 and 2 backward through time.
We use MCMC to sample tip locations, but when outputting trees sample internal node locations using a peeling algorithm as described in [22].
Thus, after the MCMC is finished we have a posterior sample of 2000 trees each tagged with estimated tip locations and internal node locations.
We post-processed each posterior tree by conducting a linear interpolation between parent-child node locations to arrive at $x$ and $y$ values at intervals of 0.05 years for each virus.
Then, for each interval, $x$ and $y$ values are averaged across the sample of posterior trees.
We draw lines between these locations to approximate mean posterior diffusion paths.
As virus lineages coalesce backwards through time down the phylogeny these diffusion paths will also coalesce.''}

It would be helpful to have phylogenetic trees for the four clades as supporting figures, since as best I understand Figure 2 and 3 do not represent the phylogenetic relationships, but only show the influence of those relationships on the antigenic map. 

\textbf{The virus phylogenies can be seen to some extent in Figures 2 and 3, i.e. phylogenetically distinct clades are apparent.  However, we agree that it would be helpful to include a cleaner picture of the virus phylogenies.  We've included a new figure (current figure 4) that shows more standard time-resolved phylogenies.  We've also included discussion of the negative correlation between antigenic drift and phylogenetic diversity.}

Also, are the phylogenetic trees built on just HA1 or the whole HA protein?

\textbf{The trees were inferred from the whole HA gene when available, but for many viruses only HA1 was available.  The phylogenetic methods available in BEAST are able to properly deal with this sort of missing data.  We've made note of this in the text.}

Although this is a stylistic rather than a substantive comment, the paper ends with a weak conclusion. The authors present a lot of complex results that could use some intuitive interpretations. Instead, their second-to-last sentence is a technical point about the phylogenetic methodology.

\textbf{We agree and have revised the conclusion accordingly.}

The authors should be commended on writing a clear Methods section, particularly as it relates to the antigenic cartography.

\textbf{Thank you. We attempted to make the cartography methods as clear as possible.}

Page 16, there is an extraneous ``the'' before the word ``observing.''

\textbf{Fixed.}

In equation 12, what is meant by the x subscript given to the sigma?

\textbf{The $\sigma_x$ in the diffusion process is describing variance among virus locations $\mathbf{X}$.  The $x$ subscript is meant to differentiate this from serum locations $\mathbf{Y}$.  This has been clarified in the text.}

%%% REVIEWER 2 %%%
\section*{Reviewer 2}

The authors extend earlier work on ``antigenic cartography'' -- that is, low-dimensional embeddings of viral strains using dimensionality reduction techniques, in order to quantify antigenic groupings of viral isolates. What is novel here compared to prior work is the introduction of a Bayesian framework for this mapping process, the observation that antigenic drift within a clade correlates with clade-specific disease incidence each season, and anti-correlates with drift in other clades. The authors also introduce a phenomenological model of antigenic variation based on diffusion, and interpret antigenic drift in terms of fitted diffusion coefficients. Finally, the authors provide antigenic maps for more clades of influenza (h3n2, h1n1, b/vic, and b/yam) than in previous publications.

Understanding antigenic drift of influenza in quantitative terms is clearly of great practical importance, and also an interesting intellectual problem in its own right. And it is nice to see antigenic maps for so many viral clades. However, I do not feel that the technical advances in ``mapping'' strains introduced here provide us much more confidence in the antigenic locations of strains than prior techniques do. Indeed, the authors themselves note (page 3) that the average predictive error of this new methods is comparable to that of Smith et all (2004). What might be more valuable, perhaps, are the results relating inferred antigenic drift and influenza incidence each season, by clade. But these results are quite possibly driven by sampling biases (see below), and I am not really convinced of their validity -- at least not without serious efforts to remove such sampling biases. Finally, the discussion of a ``diffusion'' model of drift was extremely confusing to me, and the fitted diffusion coefficients do not have a clear meaning or interpretation.

The authors do not seem to discuss this a problematic aspect of their results relating inferred drift to influenza disease incidence -- namely that biased sampling of clades, in each season, could cause the observed correlations. If clades were sampled for HI assays unequally in each season, as I suspect, then this fact alone might cause the trend towards apparently more drift in the clade that was sampled most intensively in each season (and to which most ILI was attributed). This is an obvious possible confound for the analysis presented in Fig. 4, and it should be ruled out by simulation of constant true rates of antigenic drift, variable sampling proportions in each season, and reconstructed drift using BMDS.

\textbf{We tested to see if there might be an bias in the estimates of year-to-year antigenic drift based on number of virus isolates collected by correlating correlating drift between years $i$ and $i-1$ against the ratio of the number of isolates from year $i$ to the number of isolates from year $i-1$.  We found no significant correlation (Pearson's $r = -0.01$, $p = 0.411$), suggesting our results to be unbiased with regard to sample count.  We have included this analysis in the manuscript.}

Likewise, in Fig. 5, the negative correlation between drift in different clades in each season could, again, be driven by variation in the number of strains per clade that were subjected to HI each season. The authors could rule this out either by simulation or, more simply, be repeating their analysis after subsampling an equal number of isolates per clade each season for inclusion into the BMDS.

\textbf{The above analysis applies to this correlation as well.}

I found the section fitting diffusion coefficients (in different branches) to the antigenic maps confusing -- if only because strict diffusion (in one or two dimensions) has no advection term and therefore no tendency to move in one direction or another; whereas the data clearly show an advective tendency (moving to the ``right'' over time in antigenic dimension 1). I presume the authors attempt to deal with this by imposing a prior (eq 12) for the location of a virus based on its location in a molecular phylogeny. But this would seem to conflate antigenic drift, which the authors wish to quantify using a diffusion coefficient, with genetic mutations (many of which are driven by selection for antigenic escape). In other words, this hybrid approach of assuming a prior based on a strain's genotype, and then fitting a diffusion coefficient to summarize strength of drift, does not seem to to be a pure measure of drift. What exactly the fitted diffusion coefficient means is extremely unclear to me, in any formal sense, as the procedure is so ad-hoc.

\textbf{We agree that applying the standard measure of diffusion coefficient $D$ based on displacement vs time, without accounting for advection was problematic.  We have substantial revised the text to focus on direct parameter inference of drift parameter $\drift$ and diffusion volatility parameter $\virussd$, properly separating advection and volatility in the diffusion process.  The section ``Antigenic evolution across influenza lineages'' most reflects these changes, including the addition of a new table 2.}

How do the antigenic maps shown compare to BMDS applied to pairwise aa hamming distances alone? How many serious discrepancies that are outside the confidence intervals on map reconstruction? In other words, the authors should demonstrate that the HI data actually provide substantively different results for antigenic grouping than genetic data do alone. This was an important contention of Smith et al (2004), but it was based on the position of only a handful of viruses in that paper. Do the authors have more convincing evidence in this study that the antigenic data are providing substantial extra information beyond the genetic data?

\textbf{We have revised the manuscript to include a thorough analysis of the relationship between genetic distance and antigenic distance.  This is new figure 1.  Rather than constructing BMDS maps with pairwise AA distances, we take a more direct approach and compare pairwise antigenic distance from a BMDS model using only HI data (model 2 in Table 1) and pairwise genetic distances.  We examine both pairwise AA distance and pairwise phylogenetic distance, as phylogenetic distance was the basis for our evolutionary diffusion model.  We find fairly modest correlations in most cases case (between 0.10 for B/Yam and 0.68 for A/H3N2).  Genetic data provides some predictive power for antigenic distance, but is a rather weak predictor alone.  This was a big part of our rationale for attempting a joint genetic / antigenic model.}

Minor questions:

What prior was used for the Bayesian MDS? How does this choice of prior influence the resulting antigenic maps? How have the authors justified their choice of prior?

\textbf{We assume priors on several BMDS parameters: MDS variance $\mdssd^2$, drift rate $\drift$, virus variance $\virussd^2$ and serum variance $\serumsd^2$.  We state in Materials and methods:}

\textbf{``Top-level priors for $1/\mdssd^2$, $\drift$, $1/\virussd^2$, and $1/\serumsd^2$ are assumed to follow diffuse $\mbox{Gamma}(a, b)$ distributions  with $a=0.001$ and $b=0.001$.''}

\textbf{These are extremely diffuse priors and should have very little impact on the resulting antigenic maps.  Running the analysis with improper uniform priors has almost no impact on the results.  We've made note of this in the text.}

Would not BMDS provide an entire posterior distribution for location of each strain and sera in a 2-d projection? Are authors plotting the posterior mean? What about the confidence region?

\textbf{The BMDS does give a posterior sample of virus locations.  However, summarizing this posterior distribution proved difficult.  We discussed these difficulties in Materials and Methods:}

\textbf{``There is some difficulty summarizing posterior cartographic samples, as sampled virus and serum locations represent only relative quantities, and because of this, over the course of the MCMC, virus locations may shift.
Our prior on virus and serum locations removes much of this issue, orienting the antigenic map along dimension 1 and fixing it to begin at the origin.
However, local isometries are often still a problem.
For example, in A/H3N2 the HK/68, EN/72 and VI/75 clusters may rotate in relation to other clusters.
Consequently, it may be difficult to fully align MCMC samples using Procrustes analysis.
For the present study, we take a simple approach and sample a single MCMC step and visualize the antigenic locations at this state (Figures~2, Figure~3).
Then, for specific quantities of interest, like rate of antigenic drift and rate of diffusion at different points along the phylogeny, we calculate the quantity across MCMC samples to yield an expectation and a credible interval.
This approach accurately characterizes uncertainty that may be hidden in an analysis of a single antigenic map.''}

Does a similar correlation analysis to the one shown in Fig. 4 find a relationship between estimated within-clade antigenic drift and excess P \& I mortality attributed to influenza?

\textbf{We have begun to extend the correlation analysis between within-clade antigenic drift and multiple measures of influenza incidence, including ILI, virus isolation, excess P\&I mortality and influenza-associated hospitalization rates. This analysis is also attempting to look at a finer temporal scale, rather than the course-grained 1-year bins in the current study and attempting to better match geographic bounds in antigenic and incidence data.  This said, we believe that further extending the incidence correlation requires a separate publication to  describe adequately, and is beyond the scope of the current analysis.}

The ``uninformed'' prior used in Table 1 is not described in detail, but it actually hides a subtle issue. For an unbounded variable like the x- and y- positions of a mapped isolate, the ``uninformed'' prior cannot simply be uniform on $R^2$, but rather is uniform on some compact subset of $R^2$. In other words, the authors had to assume a prior with a maximum possible x- and y-position. They don't describe what maximum bounds went into their ``uniformed'' prior, but this is well-known to influence the results of Bayesian analyses in general. This is a relatively minor issue -- authors should simply show how the form of his uniformed prior influences the resulting maps.

\textbf{We apologize for the lack of clarity.  The `Uninformed location prior' in Table 1 was described in Materials and methods as a diffuse normal prior (equation 10) with each coordinate having a variance of 10000.  We've made this clearer in the text.  We chose this variance based on prior knowledge of the scale of antigenic maps, i.e. around 50 units in breadth for A/H3N2 from Smith et al.\ 2004.  In general, our use of diffuse priors limits their influence on the resulting antigenic maps.  In this case, choosing a variance of 1000 gives very similar results.}

The authors should add a column to Table 1 showing the performance of the best possible model arising from purely genetic data -- that is, 2-D or 3-D embeddings based on aa hamming distances alone, ignoring all HI data.

\textbf{We agree that it would be ideal to provide a test error associated with a purely genetic model.  However, test errors in Table 1 are based on the prediction of a set of HI titers based on other HI titers or other HI titers plus genetic data.  Without including some form of HI information, it's difficult to see how to predict HI titer.  One could have a `distance' in terms of AA rather than antigenic map units, but in this case, one would still need to include a scaling for `serum effects', which could not be made without reference to HI data.  Regardless, we've attempted to address this issue in our discussion of correlation between antigenic and genetic distances, as described above.}

\end{document}
